src/Node.cpp

```diff
@@ -120,10 +128,15 @@ void Node::onJoyMsg(sensor_msgs::msg::Joy::SharedPtr const joy_msg)
   _robot_msg.linear.x  = (-1.0f) * joy_msg->axes[1]; /* LEFT_STICK_VERTICAL   */
   _robot_msg.angular.z =           joy_msg->axes[0]; /* LEFT_STICK_HORIZONTAL */
 
-  float const pan_angular_velocity_dps  =           joy_msg->axes[3] * get_parameter("pan_max_dps").as_double();  /* RIGHT_STICK_HORIZONTAL */
-  float const tilt_angular_velocity_dps = (-1.0f) * joy_msg->axes[4] * get_parameter("tilt_max_dps").as_double(); /* RIGHT_STICK_VERTICAL   */
-  _head_msg.angular.z = pan_angular_velocity_dps  * M_PI / 180.0f;
-  _head_msg.angular.y = tilt_angular_velocity_dps * M_PI / 180.0f;
+  /* RIGHT_STICK_HORIZONTAL */
+  auto const pan_angular_velocity =
+    (joy_msg->axes[3] * get_parameter("pan_max_dps").as_double()  * deg / s);
```

Comment:
Does `as_double` return a `double` or a `float`?  If the former, then this may have changed the underlying program slightly, so you're now using `double` instead of `float`..  If this is undesired, you could consider calling `.as_float()` (if this exists), or wrapping the `get_parameter("pan_max_dps").as_double()` part inside of `static_cast<float>(...)`.

Reply:
`as_double` returns a `double`. In this specific code the conversion isn't a problem, as its not about sub-degrees of resolution (its a physical joint of a walking robot after all). `as_float` afaik does not exist for parsing ROS parameters from launch files.